taskcluster/worker.cyml

```diff
@@ -6,3 +6,5 @@ taskcluster:
   generic:
     provisionerId: localprovisioner
     workerType: deepspeech-macos
+    # provisionerId: deepspeech-provisioner
```

Comment:
This will be switched when the proper scopes are set in https://bugzilla.mozilla.org/show_bug.cgi?id=1401596 (and the new MacBook Pro is finished to be setup)

Reply:
Looks like the bug was resolved, so this needs to be commented out and the old options removed.

Reply:
Yes, I need to finish the setup of the MacBook Pro today to be able to use that :)